src/admin-views/venue-meta-box.php

```diff
@@ -186,12 +186,18 @@
 				name  : jQuery('[name=venue\\[Venue\\]]').get(0).value
 			},
 			function (result) {
-				if (result == 1) {
+				if (jQuery('[name=venue\\[Venue\\]]').get(0).value == "") {
 					jQuery('.tribe-venue-error').remove();
+					jQuery( '[name=venue\\[Venue\\]]' ).after('<div class="tribe-venue-error error form-invalid"><?php printf( __( '%s Name Can Not be Empty', 'tribe-events-calendar' ), tribe_get_venue_label_singular() ); ?></div>');
```

Comment:
(Very minor!) please capitalize "be" to stay in titlecase:

_%s Name Can Not be Empty_ should be _%s Name Can Not Be Empty_